assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/edit.tsx

```diff
@@ -59,6 +60,7 @@ export const Edit = ( {
 						/>
 					</PanelBody>
 				) }
+				<CartSidebarCompatibilityNotice />
```

Comment:
Same here! We should place the alert under the Name & description panel before the Block's settings panels (i.e., Let's move it to line 37).